### PR TITLE
Support UNTESTED test state

### PR DESCRIPTION
**Pull request by odubaj**

### CODE REVIEW

**Review by AmsterGet — CHANGES_REQUESTED** [review]
Looks good.
Can be merged after:
- Returning the linter to the CI
- Resolving conflicts
- Backend implementation

File: Makefile [hunk 1]
```diff
@@ -42,13 +42,12 @@ fmt:
...
 
 # Builds server
-build-server: checkstyle test
+build-server: test
 	CGO_ENABLED=0 GOOS=linux $(GO) build ${BUILD_INFO_LDFLAGS} -o ${BINARY_DIR}/service-ui ./
 
 # Builds the project
 build-statics:
 	npm --prefix $(UI_BUILD_REACT) ci
-	npm --prefix $(UI_BUILD_REACT) run lint
```
[the remark this answers is not in the document]
Comment (odubaj) [reply Makefile:51]: probably just didn't wanted to run lint on each commit/build, removed only for testing reasons, will add it back

File: app/src/pages/inside/dashboardItemPage/modals/common/widgets.jsx [hunk 2]
```diff
@@ -231,7 +231,7 @@ export const getWidgets = (formatMessage) => [
       <FormattedHTMLMessage
         id={'Widgets.Description.launchStatistics'}
         defaultMessage={
-          'Shows statistics of the last launch divided into 2 sections:</br> - Skipped, Passed, Failed </br> - Product Bug, System Issue, To Investigate, Automation Bugs.'
+          'Shows statistics of the last launch divided into 2 sections:</br> - Skipped, Passed, Failed, Untested </br> - Product Bug, System Issue, To Investigate, Automation Bugs, Untested Test.'
```
[the remark this answers is not in the document]
Comment (odubaj) [reply app/src/pages/inside/dashboardItemPage/modals/common/widgets.jsx:234]: probably just a bug, will remove

**Comment by odubaj** [comment]
seems strange to me, I rebased my fork against current develop branch, then rebased UNTESTED commit according to your advices, but it does not seem to have updated this pull-request. fork to commit here: https://github.com/odubaj/service-ui/commits/develop

Maybe opening a new PR will be a silution ?

**Comment by odubaj** [comment]
Seems to be some github API issue. Everything up-to-date now.

**Comment by AmsterGet** [comment]
@odubaj Could you please resolve conflicts with the target branch?

**Comment by odubaj** [comment]
> @odubaj Could you please resolve conflicts with the target branch?

Done

**Comment by chivekrodis** [comment]
@odubaj Could you please update your fork to the latest, change target branch into `untested_test` and resolve conflicts if any?

**Comment by odubaj** [comment]
Done, available in new PR:

https://github.com/reportportal/service-ui/pull/2768

**Pull request closed without merging**